autodevops-deploy/action.yml

```diff
@@ -46,14 +51,34 @@ runs:
       env:
         NAMESPACE: ${{ steps.namespace.outputs.result }}
 
+    - name: Define environment URL
+      id: environment-url
+      shell: bash
+      run: |
+        HOST=$(cat manifests-${{ inputs.environment }}.yml | yq eval-all '.spec.rules[] .host' -)
```

Comment:
maybe add `| ("https://" + .)` at the end of `yq` expression

Reply:
good idea 😄

Reply:
La question que je me pose après coup, cela fonctionnerait-il avec un array?